### PR TITLE
Sga 88 adjust aquarium metrics based on the user s progression

### PR DESCRIPTION
**Pull request by mantasjasikenas**
- The health level of the aquarium rises upon goal achievement and declines if the goal is unmet
- The water level in the aquarium varies based on the attainment of the hydration goal

### CODE REVIEW

**Review by GillWell — APPROVED** [review]

File: app/src/main/java/com/github/k409/fitflow/data/AquariumRepository.kt [hunk 1]
```diff
@@ -15,6 +15,11 @@ import kotlinx.coroutines.tasks.await
 import javax.inject.Inject
 
 private const val AQUARIUM_COLLECTION = "aquarium"
+private const val TAG = "AquariumRepository"
+
+const val WATER_LEVEL_CHANGE_DAILY = 0.1f
```
Comment (ugnspa) [inline app/src/main/java/com/github/k409/fitflow/data/AquariumRepository.kt:20]: I suggest changing this value to 0.25 so the result of not drinking water would be more clear to the user

File: app/src/main/java/com/github/k409/fitflow/data/AquariumRepository.kt [hunk 2]
```diff
@@ -15,6 +15,11 @@ import kotlinx.coroutines.tasks.await
...
+const val HEALTH_LEVEL_CHANGE_DAILY = 0.05f
+const val HEALTH_LEVEL_CHANGE_WEEKLY = 0.25f
```
Comment (ugnspa) [inline app/src/main/java/com/github/k409/fitflow/data/AquariumRepository.kt]: change water level to 0.25f so the effect of not drinking water would be more noticeable. 
As we have 4 fish phases i suggest changing HEALTH_LEVEL_CHANGE_DAILY to 0.25f
and HEALTH_LEVEL_CHANGE_WEEKLY to 0.5f so the result of not doing tasks would be more noticeable

**Review by ugnspa — CHANGES_REQUESTED** [review]
request changes

File: app/src/main/java/com/github/k409/fitflow/data/AquariumRepository.kt (hunk 2 above)
Comment (ugnspa) [reply app/src/main/java/com/github/k409/fitflow/data/AquariumRepository.kt]: also change getPhase method in fish.kt so the values would be:
 healthLevel == 1 -> Strong
                healthLevel >= 0.5 -> Regular
                healthLevel >= 0.25 -> Dying
                else -> Dead

**Review by ugnspa — APPROVED** [review]

**Pull request merged**